dashboards-observability/public/components/event_analytics/explorer/explorer.tsx

```diff
@@ -207,11 +207,11 @@ export const Explorer = ({
   liveTailTabIdRef.current = liveTailTabId;
   liveTailNameRef.current = liveTailName;
 
+  let minInterval = 'y';
```

Comment:
I had already done the minInterval fix in a separate PR: 
https://github.com/opensearch-project/observability/pull/1298
so I don't think this is necessary anymore?

Reply:
that's probably changed from resolving conflicts, thanks for catching this.